etc/kayobe/inventory/group_vars/controllers/firewall

```diff
@@ -0,0 +1,448 @@
+firewallgen_enable_firewall: True
+
+firewallgen_ipv4_input_allow_rules:
+  - interface: "{{ internal_interface }}"
+    port: 35357
+    proto: tcp
+    destination: 10.65.1.0
+    comment: "hint: used by 'httpd' in docker container 'keystone'" 
+  - interface: "breno1.65"
```

Comment:
If you add the VIPs to the IP address mapping then this will resolve to internal_interface.